docker/run_astyle.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# Run astyle to see if there are any differences
+astyle_deb=astyle_3.0.1-1ubuntu1_amd64.deb
+wget http://archive.ubuntu.com/ubuntu/pool/universe/a/astyle/${astyle_deb}
+dpkg -i ${astyle_deb}
+rm -f ${astyle_deb}
+astyle --options=astyle_google.ini \
+       --exclude=gtest \
+       --exclude=mcnp/mcnp5/Source \
+       --exclude=mcnp/mcnp6/Source \
+       --ignore-exclude-errors \
+       --recursive \
+       --verbose \
+       --formatted \
+       "*.cc" "*.cpp" "*.h" "*.hh" "*.hpp"
+
+# Exit if astyle found diffs
+diffs=`git status --porcelain`
+if [ -z "${diffs}" ]; then
+  echo "Style guide checker passed!"
+else
+  echo "Style guide checker failed. Please run astyle."
```

Comment:
A couple small things here:

  1. Can you add ERROR (or something along those lines) at the beginning of the "Style guide checker failed..." statement? This will make it easier to find the error for those who aren't used to all that other output coming from travis.

  2. I think we should output the diff after the error message here if the test fails.

Reply:
Done and done